pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -76,6 +76,15 @@
 
 public class IngestionDelayTracker {
 
+  // Class to wrap supported timestamps collected for an ingested event
+  private static class IngestionTimestamps {
+    IngestionTimestamps(long ingestionTimesMs, long creationTimeMs) {
+      _ingestionTimeMs = ingestionTimesMs;
+      _creationTimeMs = creationTimeMs;
+    }
+    public final long _ingestionTimeMs;
+    public final long _creationTimeMs;
```

Comment:
Do these need to be public?

Reply:
yes otherwise we need getters for reading them

Reply:
Not really. If you make them package private by just dropping the `public` access modifier, you don't need to add getters for accessing the field in IngestionDelayTracker class.